spinalcordtoolbox/utils/sys.py

```diff
@@ -322,24 +322,23 @@ def run_proc(cmd, verbose=1, raise_exception=True, cwd=None, env=None, is_sct_bi
     return status, output
 
 
-def printv(string, verbose=1, type='normal'):
+def printv(string, verbose=1, type='normal', file=None):
     """
     Enables to print color-coded messages, depending on verbose status. Only use in command-line programs (e.g.,
     sct_propseg).
     """
-
-    colors = {'normal': bcolors.normal, 'info': bcolors.green, 'warning': bcolors.yellow, 'error': bcolors.red,
+    colors = {'normal': bcolors.normal, 'info': bcolors.green, 'warning': bcolors.yellow + bcolors.bold, 'error': bcolors.red + bcolors.bold,
               'code': bcolors.blue, 'bold': bcolors.bold, 'process': bcolors.magenta}
 
     if verbose:
-        # The try/except is there in case stdout does not have isatty field (it did happen to me)
+        # The try/except is there in case file does not have isatty field (it did happen to me)
         try:
             # Print color only if the output is the terminal
-            if sys.stdout.isatty():
+            if file.isatty():
```

Comment:
The default value for `file` is None, so if I understand correctly, this will always hit the `else:` case for the existing `printv` calls, thus not adding colour to anything, right?

So, shouldn't the default be `sys.stdout`?

Reply:
:sweat_smile: yes of course. Drat. I was hoping to not have to duplicate code from the stdlib.

Oh well, should be fixed now.